sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md

```diff
@@ -3,10 +3,25 @@
 ## 4.4.0-beta.4 (Unreleased)
 
 ### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+- Added new functions to key clients to enable key rotation:
+  - `KeyClient`
+    - `rotateKey(String name)`
+    - `rotateKeyWithResponse(String name, Context context)`
+    - `getKeyRotationPolicy(String name)`
+    - `getKeyRotationPolicyWithResponse(String name, Context context)`
+    - `updateKeyRotationPolicy(String name, KeyRotationPolicyProperties keyRotationPolicyProperties)`
+    - `updateKeyRotationPolicyWithResponse(String name, KeyRotationPolicyProperties keyRotationPolicyProperties, Context context)`
+  - `KeyAsyncClient`
+    - `rotateKey(String name)`
+    - `rotateKeyWithResponse(String name)`
+    - `getKeyRotationPolicy(String name)`
+    - `getKeyRotationPolicyWithResponse(String name)`
+    - `updateKeyRotationPolicy(String name, KeyRotationPolicyProperties keyRotationPolicyProperties)`
+    - `updateKeyRotationPolicyWithResponse(String name, KeyRotationPolicyProperties keyRotationPolicyProperties)`
+- Added convenience methods to create cryptography clients using key clients:
+  - `KeyClient.getCryptographyClient()`
+  - `KeyAsyncClient.getCryptographyAsyncClient()`
```

Comment:
Yeah, but I forgot to add the other changes in my previous PR so I simply added them here instead of creating a separate one exclusively for the CHANGELOG.